monitor/validators.js

```diff
@@ -89,19 +105,40 @@ async function main(bridgeMode) {
   })
   const foreignVBalances = {}
   const homeVBalances = {}
-  logger.debug('calling getGasPrices')
-  const gasPriceInGwei = await getGasPrices(GAS_PRICE_SPEED_TYPE)
-  const gasPrice = new Web3Utils.BN(Web3Utils.toWei(gasPriceInGwei.toString(10), 'gwei'))
-  const txCost = gasPrice.mul(new Web3Utils.BN(GAS_LIMIT))
+
+  logger.debug('calling home getGasPrices')
+  const homeGasPrice = await getGasPrices(
+    HOME_GAS_PRICE_ORACLE_URL,
```

Comment:
If fetching the data from the oracle fails, it will use the fallback gas price parameter. Added a special check to avoid calling `fetch` if url is not defined so there is no error on the logs. https://github.com/poanetwork/tokenbridge/pull/142/commits/b22a0a2a22de0524a47e5cad4c7aa134d021891c